build.gradle.kts

```diff
@@ -50,7 +58,3 @@ tasks {
         }
     }
 }
-
-coveralls {
```

Comment:
No longer needed, the new jacoco/coveralls plugin is made for Kotlin projects and thus knows how to pick code up properly.